.gitignore

```diff
@@ -0,0 +1,11 @@
+node_modules
+
+coverage
+
+.nyc_output
+
+.env
+
+package-lock.json
+
+.vscode
```

Comment:
Fix this new line issue before you merge

Reply:
Done.